contribute.md

```diff
@@ -0,0 +1,15 @@
+### Contribute Guideline
+Generally speaking, you need to go through the following steps to become a contributor:
+
+- Fork a repo
+- Clone the repo
+- Define a pre-commit hook
```

Comment:
```suggestion
```

Reply:
Remove this. Not required.

Reply:
Removed: Define a pre-commit hook this point. Thanks for your suggestion.